src/applications/disability-benefits/all-claims/content/ptsdWalkthroughChoice.jsx

```diff
@@ -3,18 +3,18 @@ import React from 'react';
 import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
 import { getPtsdClassification } from './ptsdClassification';
 
-export const PtsdUploadChoiceDescription = (
+export const PtsdUploadChoiceDescription = ({ formType }) => (
   <AdditionalInfo triggerText="What does this mean?">
     <h5>Continue answering questions</h5>
     <p>
       If you choose to answer questions, we’ll ask you several questions to
       learn more about your PTSD.
     </p>
-    <h5>Upload VA Form 21-0781</h5>
+    <h5>Upload VA Form {`21-0${formType}`}</h5>
     <p>
-      If you upload a completed VA Form 21-0781, we won’t ask you questions
-      about your PTSD, and you’ll move to the next section of the disability
-      application.
+      If you upload a completed VA Form {`21-0${formType}`}, we won’t ask you
+      questions about your PTSD, and you’ll move to the next section of the
+      disability application.
```

Comment:
:thinking: If they don't answer any questions or upload the form themselves, is the plan to submit an essentially blank 781/781a?

Reply:
As of now, that's essentially what they are allowed to do.

Reply:
Good to know. Thanks! :+1: